docs/source/reference/pack_management_transition.rst

```diff
@@ -0,0 +1,62 @@
+Pack Management Transition
+==========================
+
+.. warning:: The introduction of new pack management and
+  `StackStorm Exchange <https://exchange.stackstorm.org/>`__ lead to the changes that may require users
+  to update their custom :doc:`/packs`. All community packs are updated to work with |st2| 2.1
+  (and will still work with older versions of |st2|). Read on for details.
+
+
+In |st2| 2.1, pack management have received significant overhaul. With new dedicated tools, working with packs becomes very close to the "usual" package management you know from working with
+development platforms and Linux flavours. Installing, updating, and managing StackStorm packs has become a smoother, more streamlined experience.
+
+It's a big change, and you will surely enjoy the streamlined workflow for content management. By
+assuming `git`, making ``one pack - one git repo`` convention, and building tooling around it,  we
+now streamlined pack development and lifecycle. Developing a pack, contributing a fix to Community,
+checking deviation between production and upstream origins, handling versions, sharing and
+discovering your automations and integrations - all of it is becoming a breeze.
+
+However we had to make some changes that will require you to
+update your private packs, or to make or adjusting automation tools might use to
+deploy and configure your packs on |st2|.
+
+
+Good bye st2contrib, long live StackStorm Exchange
+--------------------------------------------------
+
+`StackStorm Exchange <https://exchange.stackstorm.org/>`__ is the new pack index maintained by the StackStorm team. Any pack from StackStorm Exchange can be installed with ``st2 pack install <pack>``
+in the CLI. No WebUI support yet.
```

Comment:
Is it a good idea to specifically mention something we _don't_ have? 😛 Jumps out at me a bit.